app/main.py

```diff
@@ -1,3 +1,39 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import app.customer as customer
+import app.shop as shop
+
+
+def shop_trip() -> None:
```

Comment:
In my opinion, this function is too complicated and needs refactoring. But it's up to you